charts/apisix-ingress-controller/templates/ingress-class.yaml

```diff
@@ -1,4 +1,8 @@
+{{ if (eq .Values.config.kubernetes.ingressVersion "") }}
 apiVersion: networking.k8s.io/v1
+{{- else }}
+apiVersion: {{ .Values.config.kubernetes.ingressVersion }}
```

Comment:
https://github.com/apache/apisix-ingress-controller/blob/master/conf/config-default.yaml#L72-L74

Are you sure you can use it directly?
`networking/v1beta1 != networking.k8s.io/v1beta1`

Reply:
nice catch